Explorer/Assets/DCL/Landscape/Utils/TerrainGenerationUtils.cs

```diff
@@ -0,0 +1,107 @@
+﻿using DCL.Landscape.Jobs;
+using System.Collections.Generic;
+using Unity.Collections;
+using Unity.Jobs;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace DCL.Landscape
+{
+    public static class TerrainGenerationUtils
+    {
+        public static void ExtractEmptyParcels(TerrainModel terrainModel, ref NativeArray<int2> emptyParcels, ref NativeParallelHashSet<int2> ownedParcels)
+        {
+            var tempEmptyParcels = new List<int2>();
```

Comment:
Do we need an intermediate list? Can't we use `NativeList` directly as then convert it `AsArray()` for compatibility?